dvc/utils/strictyaml.py

```diff
@@ -254,13 +265,14 @@ def validate(
     schema: Callable[[_T], _T],
     text: str = None,
     path: str = None,
+    rev: str = None,
```

Comment:
It feels wrong to extend exception to hold `rev`, but I can't think of better ways here. :(